images/daemon/image/policy.json

```diff
@@ -0,0 +1,14 @@
+{
+    "default": [
+        {
+            "type": "insecureAcceptAnything"
+        }
+    ],
+    "transports":
+        {
+            "docker-daemon":
+                {
+                    "": [{"type":"insecureAcceptAnything"}]
```

Comment:
This will probably changed in the future to signed-by right? if it did not, what impact does it have?

Reply:
I think it means we don't check the signature of container images we pull. Probably something to keep in mind for the future. I found some information here: https://docs.openshift.com/container-platform/4.6/security/container_security/security-deploy.html 

This blog describes how we could sign images. 
https://developers.redhat.com/blog/2019/10/29/verifying-signatures-of-red-hat-container-images/

Reply:
yup, I was just thinking a bit into the future a bit. What impact would insecureAccept have for us?  it would probably make sense in the future so an issue to keep track of it would make sense IMO.

Reply:
This whole `policy.json` file needs to be removed in the future. 

```go
policyContext, err := signature.NewPolicyContext(policy)
```
The content of that file can be directly initialized when we initialize the policyContext.

Reply:
created issue #30